src/components/FeatureViews/Courses/StudentEnrollment.js

```diff
@@ -0,0 +1,251 @@
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import Grid from "@material-ui/core/Grid";
+import MailOutlineIcon from "@material-ui/icons/MailOutline";
+import ChatOutlinedIcon from "@material-ui/icons/ChatOutlined";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { Link } from "react-router-dom";
+import { fullName } from "../../../utils";
+import { omouBlue, highlightColor } from "../../../theme/muiTheme";
+import SessionEmailOrNotesModal from "./SessionEmailOrNotesModal";
+
+const useStyles = makeStyles((theme) => ({
+  table: {
+    minWidth: 1460,
+    [theme.breakpoints.between("md", "lg")]: {
+      minWidth: 910,
+    },
+    [theme.breakpoints.between("sm", "md")]: {
+      minWidth: 677,
+    },
+  },
+  carrot: {
+    width: "4vw",
+  },
+  icon: {
+    width: "3vw",
+  },
+  variablesInfo: {
+    width: "15vw",
+    color: "#666666",
+    fontFamily: "Roboto",
+    fontWeight: 500,
+    fontSize: ".875rem",
+  },
+  menuSelected: {
+    "&:hover": { backgroundColor: highlightColor, color: "#28ABD5" },
+    "&:focus": { backgroundColor: highlightColor },
+  },
+  dropdown: {
+    border: "1px solid #43B5D9",
+    borderRadius: "5px",
+  },
+}));
+
+const StudentEnrollmentList = ({
+  fullStudentName,
+  accountType,
+  studentId,
+  parentAccountType,
+  parentId,
+  concatFullParentName,
+  phoneNumber,
+  handleOpenModal,
+  loggedInUserAccountType,
+}) => {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleClick = (event) => setAnchorEl(event.currentTarget);
+
+  const handleClose = () => setAnchorEl(null);
+
+  const handleOpen = (e) => {
+    e.preventDefault();
+    const currentValue = e.currentTarget.value;
+    const dataType = e.currentTarget.dataset.type;
+    handleOpenModal(currentValue, dataType);
+    setAnchorEl(null);
+  };
+
+  return (
+    <TableRow key={fullStudentName} style={{ wordBreak: "break-word" }}>
+      <TableCell
+        component="th"
+        scope="row"
+        component={Link}
+        to={`/accounts/${accountType.toLowerCase()}/${studentId}`}
+        style={{ textDecoration: "none", fontWeight: 700 }}
+      >
+        {fullStudentName}
+      </TableCell>
+      <TableCell
+        component={Link}
+        to={`/accounts/${parentAccountType.toLowerCase()}/${parentId}`}
+        style={{ textDecoration: "none" }}
+      >
+        {concatFullParentName}
+      </TableCell>
+      <TableCell>{phoneNumber}</TableCell>
+      <TableCell align="right" padding="none" size="small">
+        <Button
+          aria-controls="simple-menu"
+          aria-haspopup="true"
+          onClick={handleClick}
+        >
+          <MailOutlineIcon style={{ color: "rgb(112,105,110)" }} />
+        </Button>
+        {loggedInUserAccountType === "ADMIN" ||
+        loggedInUserAccountType === "RECEPTIONIST" ||
+        loggedInUserAccountType === "INSTRUCTOR" ? (
```

Comment:
use the access control component you should have created based on my earlier comments